src/helpers/sueHelper.ts

```diff
@@ -1,24 +1,40 @@
-import * as appJson from '../../app.json';
-import { secrets } from '../config';
+import { storageHelper } from './storageHelper';
 
-const namespace = appJson.expo.slug as keyof typeof secrets;
-export const serverUrl = secrets[namespace]?.sue?.serverUrl;
-export const apiKey = secrets[namespace]?.sue?.apiKey;
-export const jurisdictionId = secrets[namespace]?.sue?.jurisdictionId;
+export const fetchSueEndpoints = async (serviceRequestId?: number) => {
+  const globalSettings = await storageHelper.globalSettings();
+  const { apiConfig = {} } = globalSettings?.settings?.sue || {};
 
-export const sueFetchObj = {
-  method: 'GET',
-  headers: {
-    accept: 'application/json',
-    api_key: apiKey
+  if (!apiConfig) {
```

Comment:
is this possible? with the `{}` fallbacks i guess it can never be falsey/undefined. so do we need this check here? and what if undefined is returned here instead of an object, does the other codes work well?

Reply:
Yes, you are right. Even without this check, the application doesn't crash and works thanks to `{}` fallbacks. Since there is no data exchange, Network request failed error occurs and `EmptyMessage` component is displayed.